src/state/pbridge/hooks.ts

```diff
@@ -372,10 +391,135 @@ export function useBridgeSwapActionHandlers(): {
         };
       });
 
-      dispatch(setRoutes({ routes: lifiRoutes, routesLoaderStatus: false }));
+      const params: GetRoute = {
+        fromChain: fromChain?.chain_id || 1,
+        fromToken:
+          fromCurrency?.address
+            ?.toString()
+            ?.replace('0x0000000000000000000000000000000000000000', '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE') || '',
+        fromAmount: parsedAmount,
+        toChain: toChain?.chain_id || 1,
+        toToken:
+          toCurrency?.address
+            ?.toString()
+            ?.replace('0x0000000000000000000000000000000000000000', '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE') || '',
+        toAddress: recipient || fromAddress || '0x0000000000000000000000000000000000000000',
+        slippage: parseFloat(slipLimit),
+        enableForecall: true,
+      };
+
+      let squidRouteRes: RouteResponse | null;
+      try {
+        const squidRoutes = await axios.get(`${SQUID_API}/v1/route`, {
+          params: params,
+        });
+        squidRouteRes = squidRoutes.data;
+      } catch (error) {
+        squidRouteRes = null;
+      }
+
+      const squidRoute: Route | undefined = squidRouteRes?.route
+        ? {
+            nativeRoute: squidRouteRes.route as SquidRouteData,
+            bridgeType: SQUID,
+            waitingTime: calculateTransactionTime(squidRouteRes.route.estimate.estimatedRouteDuration),
```

Comment:
there are chances that `squidRouteRes` will be null, make sure you are using optional chain everywhere in this function `squidRouteRes?.route?.` etc

Reply:
It shouldn't be `null` there. Because just one line up I checked it. It shouldn't throw an error if I checked the related value in the upline.